client/data/authorizations/actions.ts

```diff
@@ -39,3 +71,58 @@ export function updateAuthorizationsSummary(
 		query,
 	};
 }
+
+export function* submitCaptureAuthorization(
+	paymentIntentId: string,
+	orderId: number
+): Generator< unknown | Authorization > {
+	try {
+		yield dispatch( STORE_NAME, 'startResolution', 'getAuthorization', [
+			paymentIntentId,
+		] );
+
+		const result = yield apiFetch( {
+			path: `/wc/v3/payments/orders/${ orderId }/capture_authorization`,
+			method: 'post',
+			data: {
+				payment_intent_id: paymentIntentId,
+			},
+		} );
+
+		const authorization = {
+			payment_intent_id: ( result as any ).id,
+			captured: ( result as any ).status === 'succeeded',
+		};
+
+		yield updateAuthorization( authorization as Authorization );
+
+		// Need to invalidate the resolution so that the components will render again.
+		yield dispatch(
+			STORE_NAME,
+			'invalidateResolutionForStoreSelector',
+			'getAuthorizations'
+		);
+
+		yield dispatch(
+			STORE_NAME,
+			'invalidateResolutionForStoreSelector',
+			'getAuthorizationsSummary'
+		);
+		// Create success notice.
+		yield dispatch(
+			'core/notices',
+			'createSuccessNotice',
+			__( 'You have captured the payment.', 'woocommerce-payments' )
+		);
+	} catch ( error ) {
+		const message = __(
+			'There has been an error capturing the payment. Please try again later.',
```

Comment:
Same as ^, I think including the order id here will help better communicate the error

Reply:
Yes, added it here [47cf00f](https://github.com/Automattic/woocommerce-payments/pull/4961/commits/47cf00fb6fece9a6a2af45a222cc90bf5b228716) thanks!